src/js/common/utils/validations.js

```diff
@@ -95,7 +95,7 @@ function isValidDateOver17(day, month, year) {
 }
 
 function isValidName(value) {
-  return /^[a-zA-Z][a-zA-Z '\-]*$/.test(value);
```

Comment:
It looked like the extra `[a-zA-Z]` was redundant? HCA was using a validation for `isValidName` without it so I removed it remove the common.